SpatialGDK/Build/Scripts/BuildWorkerConfig.bat

```diff
@@ -1,5 +1,5 @@
 @echo off
 
-pushd "%~dp0..\..\spatial"
+pushd "%~dp0..\..\..\..\..\..\spatial"
```

Comment:
😮

Reply:
Yep, this is a hack we have to have until we move to FPL